spec/README.md

```diff
@@ -0,0 +1,40 @@
+# GovStack Sandbox
+
+The GovStack Sandbox is a unified environment for all the GovStack and digital government
+ecosystem actors to discover, understand, engage, experiment, innovate, and build digital
+government solutions using the [GovStack platform](http://govstack.gitbook.io/).
+
+The sandbox is not just a playground, but rather a publicly available kit of tools, including:
+* cloud infrastructure configurations
+* container images
+* CI/CD pipeline configurations
+* Helm charts for building blocks.
+ 
+This means that you have the flexibility to take these repositories and set up your own sandbox environment.
+
+The fully operational environment offers a comprehensive API that public and private agencies can use to mimic the
+characteristics exhibited by the GovStack platform and environment on a real-time basis to help simulate responses from 
+all the systems an application interfaces with. This especially enables government agencies and digital government 
+players to experiment with innovative GovTech products or services within a well-defined space and duration. The design 
+of the GovStack sandbox encapsulates the business logic and data necessary to represent multiple GovStack capabilities 
+such as APIs, building blocks, use cases and transaction flows. This helps developers and digital government agencies 
+to achieve a more holistic view of the GovStack.
+## Get started
+
+The sandbox is reusable and replicable, which means you can set up your own GovStack sandbox within your own environment.
+
+To get more details check out the [infrastructure documentation](https://github.com/GovStackWorkingGroup/sandbox-infra). 
+
+## Candidates for building blocks
+In the Sandbox, you have the freedom to configure your own setup and run any building blocks of your choice.
+List of available BB candidates:
```

Comment:
Is this more “building blocks we have already built, which may be of help to re-use or guide your configuration of your own building block”

Reply:
Also, how will we ensure this list stays current and how can someone see a list of candidates built?

Reply:
"List of available BB candidates" is a misleading sentence.
 We currently have no official building blocks.
Here are the candidates for test purposes.

I have updated this part.